app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+from typing import Any
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            + f"Health: {self.health}, Hidden: {self.hidden}}}"
+        )
+
+    def __str__(self) -> str:
+        return (
+            "[" + ", ".join(repr(instance) for instance in Animal.alive) + "]"
+        )
```

Comment:
The `__str__` method is currently implemented as a class method. It should be an instance method to correctly represent individual object instances as strings. Consider using `self` instead of `Animal` to access instance-specific data.